src/components/pages/devices/flyouts/deviceDetails/accordion.scss

```diff
@@ -21,15 +21,16 @@
     }
 
     .accordion-toggle-btn {
+      cursor: pointer;
       background-color: transparent;
+      border: 0;
       @include rem-fallback(padding, 0px);
```

Comment:
If you are setting the value to 0 you don't need to use REMs or have a px fallback. Just use `padding: 0; margin: 0;`